src/tsconfig.spec.json

```diff
@@ -5,6 +5,7 @@
     "module": "commonjs",
     "target": "es5",
     "baseUrl": "",
+    "skipLibCheck": true, // bypass Karma type checking issues
```

Comment:
Not a huge deal but this line was highlighted in bright red in Github, I think because JSON usually doesn't allow comments, but the compiler should be able to parse this just fine.